test/main.test.js

```diff
@@ -46,6 +49,37 @@ it('reads config by direct path in environment variable', () => {
     expect(browserslist(null, { path: css })).toEqual(['ie 9', 'ie 8']);
 });
 
+it('return correct config for direct env from browserlist', () => {
+    expect(browserslist(null, { path: withEnvB, env: 'production' }))
+        .toEqual(['ie 9', 'opera 41']);
+
+    expect(browserslist(null, { path: withEnvB, env: 'development' }))
+        .toEqual(['chrome 55', 'firefox 50']);
+
+    expect(browserslist(null, { path: withEnvB, env: 'test' }))
+        .toEqual(['ie 11', 'ie 10']);
+});
+it('return correct config for direct env from package', () => {
```

Comment:
Always try to copy origin code style. I used a \n\n between tests (it is a common style)